Cargo.toml

```diff
@@ -30,12 +30,16 @@ byteorder = {version = "1", optional = true}
 multihash = {version = "0.18", optional = true}
 sqlx = {version = "0", optional = true, default-features=false}
 solana-sdk = {version = ">= 1, < 2", optional = true }
+num-bigint-dig = "0"
```

Comment:
why are these three here? Are they really needed? If so they're only needed as part of nova-tz so make them `optional = true`